bluemira/equilibria/optimisation/harmonics/toroidal_harmonics.py

```diff
@@ -0,0 +1,83 @@
+# SPDX-FileCopyrightText: 2021-present M. Coleman, J. Cook, F. Franza
```

Comment:
These functions are pretty generic and could probably live in utilities/tools.py there are already cartesian -> polar converters ~L700